### PR TITLE
Improve charting hooks

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

**Review by samshara — APPROVED** [review]

File: app/src/views/CountryProfilePreviousEvents/PastEventsChart/index.tsx [hunk 1]
```diff
@@ -109,42 +78,55 @@ function PastEventsChart(props: Props) {
...
+                                    title={chartPoint.originalData.disaster_name}
                                     description={(
                                         <>
                                             <DateOutput
-                                                value={dataPoint.originalData.date}
+                                                value={chartPoint.originalData.date}
                                                 format="yyyy MMM"
                                             />
                                             <TextOutput
+                                                // FIXME: use translations
```
Comment (barshathakuri) [inline app/src/views/CountryProfilePreviousEvents/PastEventsChart/index.tsx:103]: Please add translations.

File: app/src/views/CountryProfilePreviousEvents/PastEventsChart/index.tsx [hunk 2]
```diff
@@ -109,42 +78,55 @@ function PastEventsChart(props: Props) {
...
                                                 label="Targeted population"
-                                                value={dataPoint.originalData.targeted_population}
+                                                value={chartPoint.originalData.targeted_population}
+                                                valueType="number"
+                                            />
+                                            <TextOutput
+                                                // FIXME: use translations
```
Comment (barshathakuri) [inline app/src/views/CountryProfilePreviousEvents/PastEventsChart/index.tsx:109]: Please add translations.

File: app/src/views/CountryProfilePreviousEvents/PastEventsChart/index.tsx [hunk 3]
```diff
@@ -109,42 +78,55 @@ function PastEventsChart(props: Props) {
...
+                                                label="Amount requested"
+                                                value={chartPoint.originalData.amount_requested}
                                                 valueType="number"
                                             />
+                                            <TextOutput
+                                                // FIXME: use translations
+                                                label="Amount funded"
```
Comment (barshathakuri) [inline app/src/views/CountryProfilePreviousEvents/PastEventsChart/index.tsx:116]: Please add translations.

File: app/src/views/CountryProfileRiskWatch/RiskBarChart/CombinedChart/index.tsx [hunk 4]
```diff
@@ -351,10 +307,7 @@ function CombinedChart(props: Props) {
...
                                                 description={(
                                                     <>
-                                                        <DateOutput
-                                                            value={datum.originalData.date}
-                                                            format="yyyy MMM"
-                                                        />
+                                                        {datum.originalData.date.toLocaleDateString('default', { month: 'long' })}
                                                         {selectedRiskMetricDetail.key === 'riskScore' ? (
                                                             <TextOutput
                                                                 // FIXME: use strings
```
Comment (barshathakuri) [inline app/src/views/CountryProfileRiskWatch/RiskBarChart/CombinedChart/index.tsx:313]: Use translation

**Review by barshathakuri — APPROVED** [review]

**Review by barshathakuri — APPROVED** [review]

**Pull request merged**